graphql/resolvers.go

```diff
@@ -1160,7 +1160,7 @@ func (r *queryResolver) TaskTests(ctx context.Context, taskID string, execution
 
 	baseTestStatusMap := make(map[string]string)
 	if baseTask != nil {
-		baseTestResults, _ := r.sc.FindTestsByTaskId(baseTask.Id, "", "", "", 0, 0)
+		baseTestResults, _ := r.sc.FindTestsByTaskId(baseTask.Id, "", "", "", 0, *execution)
```

Comment:
execution is nullable so dereferencing here could cause an issue. 
we can fix this by adjusting resolver code to handle a nil execution (FindTestsByTaskId eventually uses execution in TestResultsQuery)
or we can make execution non-nullable in the taskTests query input

Reply:
I made it nullable again just to ensure backwards compatibility. It will be updating in a follow up pr after the deploys
https://github.com/evergreen-ci/evergreen/pull/4296#discussion_r556577574